deploy/e2e/eks/README.md

```diff
@@ -10,62 +10,6 @@
 
     terraform apply -var-file base.tfvars -auto-approve
 
-
 ## How to Destroy the cluster
 
-    terraform destroy -var-file base.tfvars -auto-approve  
-
-
-
-<!--- BEGIN_TF_DOCS --->
```

Comment:
I believe the terraform docs check will re-add this, unless something changed on the check that I might have missed.

Reply:
ah, this is the thing you are referring to I assume https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/34921232356fb69b5a41094246d488c37c596d97/.github/workflows/terraform-docs.yaml#L26-L33

looks like I'll have to update the `template` param so that remote and local both work for formatting docs - thanks!

Reply:
addressed in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/424/commits/19a5f13a8a21e9640bb8035be6b3d116c51fc5ee

this will most likely fail first run if merged as is (action runs on `main`, change is in PR), but should work there after